test/e2e/data/infrastructure-nutanix/v1beta1/cluster-template-failure-domains/failure-domain-patch.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: NutanixCluster
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: ${NAMESPACE}
+spec:
+  failureDomains:
+    - name: ${NUTANIX_FAILURE_DOMAIN_1_NAME}
+      controlPlane: true
```

Comment:
can we have a test with controlPlane is false so that worker node spread is also verified? i didnt find when i searched for controlPlane: false
also given there are 3 controlPlane replicas and 3 fd, so assuming each one will get one vm?

Reply:
Worker nodes will also be provisioned on FDs with control plane true. It is just that control plane node will not be scheduled on FDs that have the false value.
CAPI will indeed spread the CP nodes evenly across the 3 FDs in this scenario